dags/ods/kktix_ticket_orders/udfs/kktix_api.py

```diff
@@ -0,0 +1,127 @@
+from typing import Callable, Dict, List
+
+import requests
+import tenacity
+from airflow.hooks.http_hook import HttpHook
+from dateutil.parser import parse
+from ods.kktix_ticket_orders.udfs import kktix_loader, kktix_transformer
+
+SCHEDULE_INTERVAL_SECONDS: int = 300
+HTTP_HOOK = HttpHook(http_conn_id="kktix_api", method="GET")
+RETRY_ARGS = dict(
+    wait=tenacity.wait_none(),
+    stop=tenacity.stop_after_attempt(3),
+    retry=tenacity.retry_if_exception_type(requests.exceptions.ConnectionError),
+)
+
+
+def main(**context):
+    """
+    ETL pipeline should consists of extract, transform and load
+    """
+    schedule_interval = context["dag"].schedule_interval
+    # If we change the schedule_interval, we need to update the logic in condition_filter_callback
+    assert schedule_interval == "*/5 * * * *"
+    ts_datetime_obj = parse(context["ts"])
+    year = ts_datetime_obj.year
+    timestamp = ts_datetime_obj.timestamp()
+    event_raw_data_array = _extract(year=year, timestamp=timestamp)
+    transformed_event_raw_data_array = kktix_transformer.transform(event_raw_data_array)
+    kktix_loader.load(transformed_event_raw_data_array)
```

Comment:
the main logic is as follow:
1. extract
2. transform (do some de-identification)
4. load
    * bigquery
    * klaviyo (@henry410213028 to implement)